services/frontend/src/i18n/en_CA.json

```diff
@@ -389,8 +384,5 @@
   "work.cellphone": "Work cellphone",
   "work.location": "Location",
   "working.address": "Work address",
-  "yes": "Yes",
-  "lang.status": "Language Status",
```

Comment:
wait why did these get removed? They are used with the update I pushed.  Is the script removing more than it should